README.md

```diff
@@ -41,7 +37,7 @@ If you ran into problems, here are some of the possible sources for solutions:
 1. [Copy](http://bit.ly/subscriptionPlaylistsCopy) the Sheet to your Google Drive. 
    Afterwards you might want to update the script to the latest version of sheetScript.gs
 
-2. Setup your playlists and channels (white cells in the Sheet):
+1. Setup your playlists and channels (white cells in the Sheet):
   - Clear all existing white rows, they are just examples (i.e. delete the 3 rows).

```

Comment:
Might need a larger indent here, seems like it's resetting the ordered list count
![image](https://github.com/Elijas/auto-youtube-subscription-playlist-2/assets/17478849/8b05e315-453f-46c3-8399-e01a552ef1b9)

Reply:
Should be fixed now. Looks good there and I also noticed a couple other places it was misaligned